spec/features/navigation_spec.rb

```diff
@@ -43,4 +43,67 @@
       end
     end
   end
+
+
+  describe "as a merchant employee" do
+    it "When I click on the register link in the nav bar" do
+      merchant_1 = User.create(name: 'Bill Gates',
+                          address: '1000 Microsoft Drive',
+                          city: 'Seattle',
+                          state: 'WA',
+                          zip: '00123',
+                          email: 'bill.gates@outlook.com',
+                          password: '@%)abc123#$.',
+                          role: 1)
+
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(merchant_1)
+
+
+      visit '/items'
+
+      within 'nav' do
+        expect(page).to have_content("Merchant Dashboard")
```

Comment:
I think have_content have_link.

Reply:
I think this test is missing the "I see the same links as a regular user" part of user story 4.

Reply:
- a link to return to the welcome / home page of the application ("/")
- a link to browse all items for sale ("/items")
- a link to see all merchants ("/merchants")
- a link to my shopping cart ("/cart")
- a link to my profile page ("/profile")
- a link to log out ("/logout")